readthedocs/donate/tests.py

```diff
@@ -74,7 +97,8 @@ def test_footer(self):
             '/api/v2/footer_html/?project=pip&version=latest&page=index'
         )
         resp = json.loads(r.content)
-        self.assertEqual(resp['promo_data']['link'], '//readthedocs.org/sustainability/click/%s/%s/' % (self.promo.pk, resp['promo_data']['hash']))
+        self.assertEqual(resp['promo_data'][
+                         'link'], '//readthedocs.org/sustainability/click/%s/%s/' % (self.promo.pk, resp['promo_data']['hash']))
```

Comment:
Something odd happened here, but this should be broken up on multiple lines anyways